dask/array/ma.py

```diff
@@ -10,8 +10,8 @@
                    asanyarray, atop)
 
 
-if LooseVersion(np.__version__) < '1.11.0':
-    raise ImportError("dask.array.ma requires numpy >= 1.11.0")
+if LooseVersion(np.__version__) < '1.11.2':
+    raise ImportError("dask.array.ma requires numpy >= 1.11.2")
```

Comment:
There was something that got fixed between 1.11.1 and 1.11.2 that allows the sparse/masked chunks to work. I didn't bother diagnosing it as 1.11.2 is also several versions back, just changed the version requirement.